plenum/server/request_handlers/get_txn_handler.py

```diff
@@ -27,6 +28,12 @@ def get_result(self, request: Request):
 
         try:
             txn = self.node.getReplyFromLedger(db.ledger, seq_no)
+            audit_ledger = self.database_manager.get_ledger(AUDIT_LEDGER_ID)
+            audit_seq_no = audit_ledger.size
+            audit_txn = audit_ledger.getBySeqNo(audit_seq_no)
+            state_root = audit_txn[TXN_PAYLOAD][DATA][AUDIT_TXN_STATE_ROOT][ledger_id]
```

Comment:
Audit transaction contains only the state roots for ledger changed in this batch. So, `audit_txn[TXN_PAYLOAD][DATA][AUDIT_TXN_STATE_ROOT][ledger_id]` will raise KeyError if the ledger wasn't changed in the latest batch.

Reply:
Please add tests for 
1) Write txns to a ledger; send GET_TXN from the past for the ledger 
2) Write txns to multiple ledgers; GET_TXN from the past for different ledgers